Presentation.Web/Tests/01_Parallel/It-Contract/LocalAdmin.ItContract.UiCustomization.e2e.spec.ts

```diff
@@ -12,8 +12,15 @@ describe("Local admin is able customize the IT-Contract UI", () => {
     var testFixture = new TestFixtureWrapper();
     var navigation = new NavigationHelper();
 
+    var contractName = createName("contractName");
+    var orgName = createName("orgName");
+
     beforeAll(() => {
         testFixture.enableLongRunningTest();
```

Comment:
extend to test the tabs economy and deadlines since all settings on thos can be enabled/disabled and hence the entire tab should also be able to add/remove